integrations/docker/images/stage-2/chip-build-nuttx/Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG VERSION=1
+FROM ghcr.io/project-chip/chip-build:${VERSION}
+LABEL org.opencontainers.image.source https://github.com/project-chip/connectedhomeip
+
+RUN set -x \
+    && mv /usr/lib/python3/dist-packages/apt_pkg.cpython-38-x86_64-linux-gnu.so /usr/lib/python3/dist-packages/apt_pkg.cpython-39-x86_64-linux-gnu.so \
+    && python3.9 -m pip install --ignore-installed PyGObject \
+    && sed -i "s/isAlive/is_alive/g" /usr/lib/python3/dist-packages/softwareproperties/SoftwareProperties.py \
+    && echo -e '\n' | add-apt-repository ppa:ubuntu-toolchain-r/test \
+    && echo -n "Y" | apt install g++-13-multilib \
```

Comment:
using apt-install and setting alternative would make it hard to include this in the vscode image. I am thinking we could:

- update gcc in the main chip-build since we use gcc for linux host builds so it should be fine. This will make chip-build-vscode pull it automatically
- use some stand-alone tar.gz/tar.xz and unpack into /opt, so that vscode image can create its own copy

Reply:
@andy31415 Do you mean to move these changes to integrations/docker/images/base/chip-build/Dockerfile file?